pkg/ubp/storage.go

```diff
@@ -31,11 +27,17 @@ func (s StorageType) Size() (ByteSize, error) {
 
 var ErrInvalidStorageType = errors.New("invalid storage type")
 
+// StorageTypeDefault is used when no storage type was specified. This can only happen
+// for a converted blueprint, UBP schema requires a storage type to be specified.
+const StorageTypeDefault StorageType = ""
+
 func ParseStorageType(s string) (StorageType, error) {
 	switch strings.ToLower(s) {
-	case "gpt", "":
+	case "":
+		return StorageTypeDefault, nil
+	case "gpt":
 		return StorageTypeGPT, nil
-	case "mbr":
+	case "mbr", "dos":
```

Comment:
Do we have a test that covers storage type "dos" ? This PR seems to replace all "dos"->"mbr", is there some testcasae left for "dos"?

Reply:
Yes we have. It was the reason why I actually made the change - I noticed this use case coming from `images`:

https://github.com/osbuild/blueprint-schema/blob/993ad4c66cdeb043417012bf2238806002d8be95/testdata/bp-partitioning-dos-lvm.in.json#L4

It is a copy of:

https://github.com/osbuild/images/blob/main/test/configs/partitioning-dos-lvm.json#L6

There is a makefile target to copy all test cases from `images` project:

https://github.com/osbuild/blueprint-schema/blob/993ad4c66cdeb043417012bf2238806002d8be95/Makefile#L99